v1.1/create-security-certificates-openssl.md

```diff
@@ -0,0 +1,326 @@
+    req_extensions = extensions
+
+    [ distinguished_name ]
+    organizationName = Cockroach
+    commonName = node
+
+    [ extensions ]
+    subjectAltName = DNS:<node-hostname>,DNS:<node-domain>,IP:<IP Address>
+    ~~~
+
+    {{site.data.alerts.callout_info}}The <code>commonName</code> and <code>subjectAltName</code> parameters are vital for CockroachDB functions. It is also important that <code>commonName</code> be set to <code>node</code>. You can modify or omit other parameters as per your preferred OpenSSL configuration, but do not omit the <code>commonName</code> and <code>subjectAltName</code> parameters.  {{site.data.alerts.end}}
+  
+2. Create the key for the first node using the [`openssl genrsa`](https://wiki.openssl.org/index.php/Manual:Genrsa(1)) command:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ openssl genrsa -out certs/node.key 2048 
+    ~~~
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ chmod 400 certs/node.key
+    ~~~
+
+3. Create the CSR for the first node using the [`openssl req`](https://wiki.openssl.org/index.php/Manual:Req(1)) command:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    # Create Node certificate signing request.
+    $ openssl req \
+      -new \
+      -config node.cnf \
+      -key certs/node.key \
+      -out node.csr \
+      -batch
+    ~~~
+
+4. Sign the node CSR to create the node certificate for the first node using the [`openssl ca`](https://wiki.openssl.org/index.php/Manual:Ca(1)) command. 
+
+    You can set the node certificate expiration period using the `days` flag. We recommend using the CockroachDB default value of the node certificate expiration period, which is 1830 days.
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    # Sign the CSR using the CA key.
+    $ openssl ca \
+      -config ca.cnf \
+      -keyfile my-safe-directory/ca.key \
+      -cert certs/ca.crt \
+      -policy signing_policy \
+      -extensions signing_node_req \
+      -out certs/node.crt \
+      -outdir certs/ \
+      -in node.csr \
+      -days 1830 \
+      -batch
+    ~~~
+
+5. Upload certificates to the first node:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    # Create the certs directory:
+    $ ssh <username>@<node1 address> "mkdir certs"
+    ~~~
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    # Upload the CA certificate and node certificate and key:
+    $ scp certs/ca.crt \
+    certs/node.crt \
+    certs/node.key \
+    <username>@<node1 address>:~/certs
+    ~~~
+
+6. Delete the local copy of the first node's certificate and key:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ rm certs/node.crt certs/node.key
+    ~~~
+
+    {{site.data.alerts.callout_info}}This is necessary because the certificates and keys for additional nodes will also be named <code>node.crt</code> and <code>node.key</code>.{{site.data.alerts.end}}
+
+7. Repeat steps 1 - 6 for each additional node.
```

Comment:
Marc and I realized that openssl creates `.pem` files that are exact duplicates of the `.crt` files. So let's add a step `8`:

> 8. Remove the `.pem` files in the `certs`. These files are unnecessary duplicates of the `.crt` files that CockroachDB requires.